packages/react/src/themes/colorUtils.ts

```diff
@@ -16,3 +21,23 @@ export const extendColorScheme = (
   })
   return result
 }
+
+export function pickValuesFromColorScheme<T extends Partial<ComponentAreaName>>(
```

Comment:
maybe simpler `pickColors` name, given that arguments are descriptive enough?

Reply:
Hmm, it's not about picking colors is more about picking areas (like foreground, background)... So maybe `pickColorAreas` would be better?